tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -330,7 +330,19 @@ def get_cpu_extension(cpu_extensions, selection_mode):
         if cpu_extensions_name != 'AUTO':
             return cpu_extensions
         extensions_path = cpu_extensions.parent
-        file_format = '{}.dll' if platform.system() == 'Windows' else 'lib{}.so'
+        os_specific_formats = {
+            'Windows': '{}.dll',
+            'Darwin': 'lib{}d.dylib',
```

Comment:
This should be `lib{}.dylib`. The `d` version is the debug build, I believe.

Also, I would suggest to list these in alphabetical order and to add a trailing comma.

Reply:
> The d version is the debug build, I believe.
Most likely. I also add additional search loop if a release lib is not found.

Reply:
> Most likely. I also add additional search loop if a release lib is not found.

Why would it not be found?

Reply:
Possibly, if a user builds IE from sources in debug mode, he can not have release lib (but I do not sure about it...)

Reply:
Fair enough. However, macOS support should go into R2, while I would prefer to postpone the debug mode support until R3, to minimize the potential for regressions. So let's stick with just enabling macOS in this PR, OK?

Reply:
(FWIW, I don't think searching for either version of the library makes sense anyway. The extension library should match the main IE library.)

Reply:
Ok, I remove debug.